doc/Architecture/arc-cs_x-road_central_server_architecture.md

```diff
@@ -114,49 +116,52 @@ See X-Road terms and abbreviations documentation \[[TA-TERMS](#Ref_TERMS)\].
 
+11. <a id="Ref_OPENAPI" class="anchor"></a>\[OPENAPI\] OpenAPI Specification 3.0.0. 26th July 2017.
+
+12. <a id="Ref_ADMIN-SERVICE-API" class="anchor"></a>\[ADMIN-SERVICE-API\] X-Road Central Server Admin API OpenAPI Specification, <https://github.com/nordic-institute/X-Road/blob/develop/src/central-server/openapi-model/src/main/resources/openapi-definition.yaml>.
+
 ## 2 Component View
 
 [Figure 1](#Ref_Components_and_interfaces_of_the_X_Road_central_server) shows the main components and interfaces of the X-Road central server. The components and the interfaces are described in detail in the following sections.
 
 
 <a id="Ref_Components_and_interfaces_of_the_X_Road_central_server" class="anchor"></a>
-![](img/arc-cs_components_and_interfaces_of_the_x_road_central_server.png)
+![](img/arc-cs_components_and_interfaces_of_the_x_road_central_server.svg)
 
 Figure 1. Components and interfaces of the X-Road central server
 
 Technologies used in the central server can be found here: \[[ARC-TEC](#Ref_ARC-TEC)\]
 
+### 2.1 Administration Service
 
-### 2.1 Web Server
+Central Server Administration Service allows management of X-Road members and security servers and define the global configuration parameters that are distributed to the security servers. 
 
-The global configuration files generated by the central server need to be made available to configuration clients. The HTTP-based protocol used for downloading configuration is described in \[[PR-GCONF](#Ref_PR-GCONF)\]. Technically, the configuration consists of a set of files that are shared out using standard web server (nginx\[[1](#Ref_1)\] web server is used). The global configuration generation is handled by the management services component.
+User action events that change the system state or configuration are logged into the audit log. The actions are logged regardless of whether the outcome was a success or a failure. The complete list of the audit log events is described in \[[SPEC-AL](#Ref_SPEC-AL)\].
 
+Administration Service is a Spring Boot\[[1](#Ref_1)\] application with embedded Tomcat servlet container.
 
 <a id="Ref_1" class="anchor"></a>
-\[1\] See <http://nginx.org/> for details.
-
-
-### 2.2 Management Services
+\[1\] See <https://spring.io/projects/spring-boot> for details.
 
-A collection of services including:
+### 2.2 Web Server
 
--   management services that are used by security servers to perform management tasks such as registering a security server client or deleting an authentication certificate;
+The global configuration files generated by the central server need to be made available to configuration clients. The HTTP-based protocol used for downloading configuration is described in \[[PR-GCONF](#Ref_PR-GCONF)\]. Technically, the configuration consists of a set of files that are shared out using standard web server (nginx\[[2](#Ref_2)\] web server is used). The global configuration generation is handled by the management services component.
```

Comment:
Central Server and Security Server should be written with capital initial letters throughout the document.